src/app/Providers/LdapEmulatorServiceProvider.php

```diff
@@ -37,9 +37,10 @@ public static function setActingUser(string $username)
         $ldapUsernameKey = config('ldap-emulator.ldap-username-key');
         $model = config('ldap-emulator.ldap-user-model');
 
-        Container::getDefaultConnection()->actingAs(
-            $model::findBy($ldapUsernameKey, strtolower($username)),
-        );
+        $user =  $model::findBy($ldapUsernameKey, strtolower($username));
+        if ($user !== null) {
```

Comment:
What happens if you throw an authentication exception at this point?

Reply:
I will give this a go on the Home laravel 11 branch

Reply:
I have put and AuthenticationException in the body of the if statement and the testPutsEmailInSessionOnSuccess and testRedirectsOnSuccess fails. I tried a few variations, but no joy